packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -349,12 +349,12 @@ class AndroidMotionEvent {
   /// See Android's [MotionEvent#getDeviceId](https://developer.android.com/reference/android/view/MotionEvent.html#getDeviceId()).
   final int deviceId;
 
-  /// A bitfield indicating which edges, if any, were touched by this MotionEvent.
+  /// A bit field indicating which edges, if any, were touched by this MotionEvent.
   ///
   /// See Android's [MotionEvent#getEdgeFlags](https://developer.android.com/reference/android/view/MotionEvent.html#getEdgeFlags()).
   final int edgeFlags;
 
-  /// The source of this event (e.g a touchpad or stylus).
+  /// The source of this event (e.g a touch pad or stylus).
```

Comment:
IMO we should leave this one alone.
https://www.dictionary.com/browse/touchpad?s=t
https://en.wikipedia.org/wiki/Touchpad

On my Linux machine, the configuration for this is under "Mouse and Touchpad".

Reply:
OK, reverted.